eslint.config.mjs

```diff
@@ -211,6 +211,7 @@ export default tseslint.config(
       'one-var': ['error', 'never'],
       'prefer-arrow-callback': 'error',
       'prefer-rest-params': 'error',
+      radix: 'error',
```

Comment:
Huh, from reading https://eslint.org/docs/latest/rules/radix:

> On the other hand, if the code is targeting only ES5-compliant environments passing the radix `10` may be redundant. In such a case you might want to disallow using such a radix.

Maybe we don't need this?

Reply:
`parseInt` still auto-detects `0x`, so I'd say this is still nice to have, especially when this PR is moving away from `parseInt` to `Number`, which I vastly prefer.